lib/vsc/filesystem/operator.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: latin-1 -*-
+#
+# Copyright 2009-2022 Ghent University
+#
+# This file is part of vsc-filesystems,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# the Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/hpcugent/vsc-filesystems
+#
+# All rights reserved.
+#
+"""
+Interface to dynamically load vsc.filesystem modules and instantiate its XxxxOperations class
+
+@author: Alex Domingo (Vrije Universiteit Brussel)
+"""
+import importlib
+import logging
+
+STORAGE_OPERATORS = {
+    'posix': ('PosixOperations', 'PosixOperationError'),
```

Comment:
you can derive the values from the key, or at least 

```
posix : Posix
```
(as oceanstor is a bit more then just `.capitalise`

Reply:
ok, I settled in using a simple tuple with the storage labels used in the `XxxxOperations` classes and define a couple globals with the suffix of each type of class. See https://github.com/hpcugent/vsc-filesystems/pull/83/commits/329e4112e1e5545a1e2e98346920799c7aaa7231